src/home/substatus.asm

```diff
@@ -580,7 +583,7 @@ GetLoadedCard1RetreatCost::
 	call GetTurnDuelistVariable
 .check_bench_loop
 	ld a, [hli]
-	cp -1
+	cp $ff
```

Comment:
I would probably leave these as `-1` unless you have a compelling reason to use $ff.

Reply:
No, I only changed those to be more consistent with what's used in the rest of the disassembly. however, after double-checking the style guide, it does look like `-1` is the preferred option for empty slots, so I'll change those back.